scheduler/graphql_mid/schema.py

```diff
@@ -1,27 +1,31 @@
 # Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
 # For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
-
+import uuid
 from typing import List
 import strawberry # noqa
 from astropy.time import Time
-from lucupy.minimodel import Site, ALL_SITES, NightIndex
+from lucupy.minimodel import Site, ALL_SITES, NightIndex, Conditions, CloudCover, SkyBackground, WaterVapor, \
+    ImageQuality
 
 from scheduler.core.service.service import Service
 from scheduler.core.sources import Services, Sources
 from scheduler.core.builder.modes import SchedulerModes
-from scheduler.core.eventsqueue import EventQueue
+from scheduler.core.eventsqueue import EventQueue, WeatherChange
 from scheduler.db.planmanager import PlanManager
 
 
 from .types import (SPlans, NewNightPlans, ChangeOriginSuccess,
-                    SourceFileHandlerResponse, SNightTimelines)
-from .inputs import CreateNewScheduleInput, UseFilesSourceInput
+                    SourceFileHandlerResponse, SNightTimelines, EventsAddedSuccess, EventsAddedError,
+                    EventsAddedResponse, EventRemovedResponse)
+from .inputs import CreateNewScheduleInput, UseFilesSourceInput, NewWeatherChangeInput
 from .scalars import SOrigin
-
+from ..config import config
+from ..core.time import TimeHandler
 
 # TODO: This variables need a Redis cache to work with different mutation correctly
 sources = Sources()
 event_queue = EventQueue(frozenset([NightIndex(i) for i in range(3)]), ALL_SITES)
+time_handler = TimeHandler(start=None, num_nights_to_schedule=None, time_slot_length=config.collector.time_slot_length)
```

Comment:
Here you set the `TimeHandler` start to `None`... won't this cause an exception when you use the `TimeHandler` methods?